docs/install/1-install-server.rst

```diff
@@ -433,11 +413,10 @@ ssh and if the remote agent is used.
 
     sudo chown inmanta:inmanta /var/lib/inmanta/.ssh/config
 
-3. Add the public key to any git repositories and save it to include in configuration models that require remote agents.
-4. Test if you can login into a machine that has the public key and make sure ssh does not show you any prompts to store
+3. Test if you can login into a machine that has the public key and make sure ssh does not show you any prompts to store
```

Comment:
Shouldn't we say here "try to clone git repos", instead of "try to login to machines"?